notifier/notifier.go

```diff
@@ -30,21 +30,22 @@ import (
 
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/go-openapi/strfmt"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	config_util "github.com/prometheus/common/config"
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/common/version"
 
+	"github.com/prometheus/alertmanager/api/v2/models"
```

Comment:
This patch vendors the Alertmanager API v2 model. From my point of view, this gives us a strong contract between the two APIs. Let me know what you think.

Reply:
Looks good to me.